spec/javascript/components/atoms/Button.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import Button from '~/components/atoms/Button'
+
+describe('Button', () => {
+  test('it renders a button with the correct default properties', () => {
+    const wrapper = shallow(<Button text='This is a button' />)
+    const selector = 'button'
+    expect(wrapper.find(selector).exists()).toBeTruthy()
```

Comment:
nit: prefer `expect(wrapper).toHaveLength(1)` over `expect(wrapper.exists()).toBeTruthy()`

Mostly for consistency but also so that it fails if there are 2 or more.

Reply:
good idea, thanks!